### ISSUE TITLE
InterceptorPB not connected

### ISSUE THREAD

**Issue by kimseasok** [jookies/jasmin#941]
I've been trying to add an interceptor for the MTRouter.
However, there is no luck. I encountered with the bellow message.

```
[2021-01-11 05:01:53,225: ERROR/ForkPoolWorker-1] [7c45f0eb-3970-4b76-b8c1-1c5761c0e51f] Error "InterceptorPB not connected !
[2021-01-11 05:01:53,227: INFO/ForkPoolWorker-1] Task jasmin.protocols.rest.tasks.httpapi_send[2cdecbef-eee2-42b8-a2ae-a13177a6f494] succeeded in 0.0162822290004s: None
```

Then, I manually start the interceptor with the following command.

```
root@bc22beb7b318:/# jasmind.py --username jcliadmin --password jclipwd --enable-interceptor-client
/usr/local/lib/python2.7/dist-packages/OpenSSL/crypto.py:14: CryptographyDeprecationWarning: Python 2 is no longer supported by the Python core team. Support for it is now deprecated in cryptography, and will be removed in the next release.
  from cryptography import utils, x509
Lock not acquired ! exiting
```
Here is the interceptor log

```
root@bc22beb7b318:/# cat /var/log/jasmin/interceptor.log 
2021-01-09 13:40:43 INFO     6 Interceptor configured and ready.
2021-01-09 13:40:52 INFO     6 Authenticated Avatar: iadmin
root@bc22beb7b318:/# 
```
Is there any specific configuration I can do to make the interceptor work?

Note: Redis and RabbitMQ are running in docker container. I changed the jasmin.cfg and pointed to the containers.

Thanks~

**Comment by farirat** [comment]
You may try to raise your logging level, switch to py3 version for better support (jasmin 0.10.x).

**Comment by kimseasok** [comment]
> You may try to raise your logging level, switch to py3 version for better support (jasmin 0.10.x).

Thanks, I've tried the py3, but it not current convenient for us.
We plan to switch ASAP.

**Comment by kimseasok** [comment]
The issue solved after upgrade to 9.36
